.github/workflows/actions/deploy-environment/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
   pr_id:
     description: "Pull Request number"
     required: false
+  platform:
+    description: "PaaS or AKS"
```

Comment:
Can we use lowercase in the description as it's what is expected in the code?

Reply:
Yeah, no problem!